compiler-rt/lib/scudo/standalone/tests/secondary_test.cpp

```diff
@@ -265,3 +265,99 @@ TEST_F(MapAllocatorWithReleaseTest, SecondaryThreadsRace) {
   Allocator->getStats(&Str);
   Str.output();
 }
+
+struct MapAllocatorCacheTest : public Test {
+  static constexpr scudo::u32 MarkerBytes = 0xDEADBEEF;
+
+  static void testUnmapCallback(scudo::MemMapT &MemMap) {
+    scudo::u32 *Ptr = reinterpret_cast<scudo::u32 *>(MemMap.getBase());
+    *Ptr = MarkerBytes;
+  }
+
+  using Config = scudo::DefaultConfig;
```

Comment:
It doesn't look like this is used anywhere.

Reply:
`unmapCallback` or `Config`? `unmapCallback` is used in the definition of `CacheT` and `Config` is used to define `SecondaryConfig`.

Reply:
I would use the value directly then.

However, is there a reason you are using the DefaultConfig? I am always worried about using something that might change and cause weird failurs.

Reply:
Do you mean use of `Config` directly?

I used `DefaultConfig` just because I assumed we would want to test it on the default values. There aren't any specific configuration requirements for this test fixture so if I set up a test config I wouldn't have any justification for using different values than the `DefaultConfig`.

Reply:
What if the DefaultConfig changes to not use a cached secondary? Would that cause this to behave incorrectly?

Reply:
I see. I suppose the one's specific to cache order would likely not work. I noticed that all of the other test fixtures use `DefaultConfig` but perhaps they don't rely on the cache implementation, and I know we are trying to make this test more independent of the allocator implementation than the previous tests. Do you think that for a test config I can use the current cache config in `DefaultConfig`?

Reply:
Given that the test doesn't depend on `MapAllocator`, maybe we can consider just having one specific config locally for your tests?

Reply:
I think we would need to set up a new config wrapper, though, because the `CacheConfig` is currently templated by an allocator configuration. I just realized `secondary_test.cpp` already has a test config within this same file that deliberately disables the primary allocator, so I think I can just use this instead of setting up a new config.

Reply:
Discussed in meeting: we'll use the `TestConfig` already set up in `secondary_test.cpp`